src/board/leds.c

```diff
@@ -26,8 +25,8 @@
 /** Setup the LED GPIOs. */
 void led_setup(void)
 {
-  RCC_AHB1ENR |= RCC_AHB1ENR_IOPCEN;
-  gpio_mode_setup(GPIOC, GPIO_MODE_OUTPUT, GPIO_PUPD_NONE, GPIO3 | GPIO4);
+  palSetPadMode(GPIOC, GPIOC_LED1, PAL_MODE_OUTPUT_PUSHPULL);
```

Comment:
Ok.  I agree this is pretty janky.  I'm busy with the port to ChibiOS 3 now.  If that will clean it up we can wait for those changes.  Actually this code can go, as all the ports/pads are initialized in halInit() anyway, to the values in board.h.  It's probably cleaner to have it only done in one place.